lib/LedgerSMB/I18N.pm

```diff
@@ -127,11 +126,12 @@ Get a country localized list to allow user selection
 
 sub get_country_list {
     my $language = shift;
-    my %regions = Locale::CLDR->new($language)->all_regions->%*;
+    my $locale = Locales->new($language);
+    my %regions = $locale->get_territory_lookup();
     return [
         sort { $a->{text} cmp $b->{text} }
         map { +{ value => uc($_),
-                 text  => $regions{$_} }
+                 text  => $regions{lc($_)} }
```

Comment:
When iterating over the keys of `%regions`, I'd expect you shouldn't be transforming the key before doing a lookup (i.e. no `lc()`)?

Reply:
The keys are already lowercased but I fixed it.